File: bootcamp_c_cpp_dsa-main/bootcamp_c_cpp_dsa-main/C/magic2/p9.c
#include<stdio.h>
void main(){

	int arr1[]={10,20,30,40,50};
	int arr2[]={60,70,80,90,100};

	int *iptr1=null;
	int *iptr2=null;

	iptr1=arr1+3;
	iptr2=arr2+2;

	iptr1=35;

	for(int i=0;i<5;i++){

		printf("%d",arr1[i]);
	}
	for(int j=0;j<5;j++){
		printf("%d",arr2[i]);
	}
}

File: java-main/java-main/Array/Ex2/p9.java
//Write a Java program to merge two given arrays.


class Demo{
	public static void main(String[]args){
		int arr1[]={10,20,30,40,50};
		int arr2[]={60,70,80,90};
		int arr3[]=new int [arr1.length+arr2.length];
		int i=0;
		for(;i<arr1.length;i++){
			arr3[i]=arr1[i];
		}
		for(int j=0;j<arr2.length;j++,i++){
			arr3[i]=arr2[j];
		}
		for(int k=0;k<arr3.length;k++){
			System.out.println(arr3[k]+ " ");
		}
	}
}
